app/controllers/users_controller.rb

```diff
@@ -22,9 +25,50 @@ def create
     end
   end
 
+  def edit
+    @user = User.find_by(id: params[:id])
```

Comment:
thay vì phải khai báo lặp lại ở mỗi action thì tạo 1 method riêng rồi goi ở call_back e nhé
```suggestion
before_action :load_user, only: %i(...)

def edit; end

private

def load_user
    @user = User.find_by(id: params[:id])
    unless @user
      flash ... # báo lỗi ko tìm thấy
      reddirect_to ...
    end
end

def correct_user
  load_user
  ...
end
```